packages/appstore/src/components/cfds-listing/index.tsx

```diff
@@ -159,12 +161,12 @@ const CFDsListing = observer(() => {
     return (
         <ListingContainer
             title={
-                !isMobile() && (
+                isDesktop && (
                     <div className='cfd-accounts__title'>
                         <Text size='sm' weight='bold' color='prominent'>
                             {localize('CFDs')}
                         </Text>
-                        <CompareAccount accounts_sub_text={accounts_sub_text} is_desktop={!isMobile()} />
+                        <CompareAccount accounts_sub_text={accounts_sub_text} is_desktop={isDesktop} />
```

Comment:
If `isDesktop === true`, you'll get to line 169 anyway, so we can just left ` <CompareAccount accounts_sub_text={accounts_sub_text} is_desktop />`